### ISSUE TITLE
Dependencies specification?

### ISSUE THREAD

**Issue by karadaharu** [althonos/InstaLooter#96]
Thank you for the good tool.

When I installed InstaLooter using pip with a bit old libraries, I got an error which was not easy to debug : 

```
$ python --version
Python 3.5.2 :: Anaconda 2.4.0 (x86_64)
$ pip list --format=columns| grep 'beautiful\|docopt\|progress\|requests \|six\|dateutil\|hues'
beautifulsoup4              4.6.0
docopt                      0.6.2
hues                        0.2.2
progressbar2                3.34.2
python-dateutil             2.6.0
requests                    2.14.2
six                         1.10.0
$ instaLooter post "https://www.instagram.com/p/BFB6znLg5s1/" .
15:20:21 - ERROR - __exit__
```

And when I updated `requests` package from 2.14.2 to 2.18.4, it worked well.

If you think it is not good, could you specify version dependencies in `setup.py` or write some information on the Installation section of README? I know you already write requirements on the top of README, but I think it is more user-friendly if you do either of them, and I can help you if you want.

**Comment by althonos** [comment]
Yes, I did not pin dependencies properly because I am a bit lazy ;)

I'll try to find a working configuration based on your version specifiers, but indeed the error was caused by `requests` not implementing the context manager interface for `Response` objects prior to `2.18.0`.

**Comment by althonos** [comment]
So here's the minimum working version I ended up with:
```
    six ~= 1.4.0
    beautifulsoup4 ~= 4.4.0
    progressbar2 ~= 3.1.0
    requests ~= 2.18.0
    docopt ~= 0.4.0
    python-dateutil ~= 2.1.0
    hues ~= 0.2.2
```

I'll release a bugfix version with the new version specifiers.

**Comment by althonos** [comment]
Fixed in `v0.13.1`
